packages/lit-html/src/lit-html.ts

```diff
@@ -366,17 +371,21 @@ class Template {
             regex = comment2Regex;
           } else if (match[2] !== undefined) {
             if (rawTextElement.test(match[2])) {
-              regex = new RegExp(`<\/${match[2]}`, 'g');
-            } else {
-              regex = tagRegex;
+              // Record if we encounter a raw-text element. We'll switch to
+              // this regex at the end of the tag
+              rawTextTag = new RegExp(`<\/${match[2]}`, 'g');
```

Comment:
This change seems to imply that you could see the end of the tag without actually being at the end. What case is that? Maybe put an example in the comment?

Reply:
OK, `rawTextTag` indicates we are inside a script|style|textarea and moving right to this would skip parsing the attributes inside the tag itself.